01_fib/alanhillgemann/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+	if n < 0 {
+		log.SetFlags(0)
```

Comment:
Initialize logging in main I would say. I'd also find it more intuitive to read: `log.SetFlags(log.Ldate)`.

Reply:
Fixed